### DISCUSSION TITLE
Android text to speech

### DISCUSSION THREAD

**Discussion by ghost** [godot-extended-libraries/godot-ideas#41] [Ideas]
It is important to add a text to speech plugin in godot. I think this is important because
When a dialogue shows the user can hear the speech . I saw this feature in many games. I think this is essential for reducing app size.

**Comment by Calinou** [comment]
Support for text-to-speech is being implemented by https://github.com/godotengine/godot/pull/56192.

Either way, this discussion is a duplicate of https://github.com/godotengine/godot-proposals/issues/3584 :slightly_smiling_face:
